libnavigator/src/test/java/com/mapbox/navigation/navigator/ActiveGuidanceOptionsMapperTest.kt

```diff
@@ -13,93 +12,6 @@ import org.junit.Test
 
 class ActiveGuidanceOptionsMapperTest {
 
-    // TODO: Mocking routeOptions.coordinates() to emptyList()
-    //  Tests need to be updated / new ones need to be added in https://github.com/mapbox/mapbox-navigation-android/pull/3581
-
```

Comment:
Assuming all the tests removed from here are covered as part of the `Parameterized` new ones added, right?

Reply:
Yeah, exactly 💯